### DISCUSSION TITLE
Battle Idea

### DISCUSSION THREAD

**Discussion by rafehimself** [cssbattle/forum#205] [Ideas]
I came up with idea for a battle connected with recent launch of the most advanced telescope in human history called James Webb Telescope 🚀 🧑‍🚀 ⭐ ⚫ 🔭 🌍 🛰️ 📡 
name might be Space - important aspect of css as well 😉 
targets general ideas
![image](https://user-images.githubusercontent.com/58576132/147605381-15db9c81-1169-46fd-a246-9ea0d9e77853.png)

that's just general concepts that needs to be beautify and accessible

**Comment by chinchang** [comment]
oh woww!! This is really great idea @rafehimself ! 🤩 We'll definitely gonna keep this in mind for next battles!

**Comment by klauseba** [comment]
I also have a battle idea> Mr Oizo logo (if it's legal to draw it)
![3](https://github.com/cssbattle/forum/assets/66975395/246ae1d0-783f-4f65-81f5-399682793085)
